tests/integration/test_litbase.c

```diff
@@ -0,0 +1,77 @@
+// This sample code demonstrates the option CS_OPT_LITBASE
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <capstone/platform.h>
+#include <capstone/capstone.h>
+
+#define DATA "\x11\x00\x00\x11\xff\xff"
+
+static void print_string_hex(unsigned char *str, size_t len)
+{
+	unsigned char *c;
+
+	for (c = str; c < str + len; c++) {
+		printf("%02x ", *c & 0xff);
+	}
+	printf("\t");
+}
+
+static void print_insn(csh handle)
+{
+	cs_insn *insn;
+	size_t count;
+
+	count = cs_disasm(handle, (const uint8_t *)DATA, sizeof(DATA) - 1,
+			  0x10000, 2, &insn);
+	if (count) {
+		for (int i = 0; i < count; ++i) {
+			print_string_hex((unsigned char *)DATA,
+					 sizeof(DATA) - 1);
+			printf("\t%s\t%s\n", insn[i].mnemonic, insn[i].op_str);
```

Comment:
Just printing is not enough :D
Please compare the output to a the expected asm text. And fail with an error message if it doesn't match.